packages/fxa-auth-server/test/local/routes/account.js

```diff
@@ -1598,10 +1604,24 @@ describe('/account/destroy', () => {
   it('should delete the account', () => {
     const email = 'foo@example.com';
     const uid = uuid.v4('binary').toString('hex');
+    const expectedSubscriptions = [
+      { uid, subscriptionId: '123' },
+      { uid, subscriptionId: '456' },
+      { uid, subscriptionId: '789' }
+    ];
     const mockDB = mocks.mockDB({
       email: email,
       uid: uid
     });
+    mockDB.fetchAccountSubscriptions =
+      sinon.spy(async (uid) => expectedSubscriptions);
+    mockDB.deleteAccountSubscription =
+      sinon.spy(async (uid, subscriptionId) => true);
```

Comment:
You're welcome to add these to the method list for `mockDB` if you like, to save defining them inline in the test body like this. I realise they're not the most broadly useful methods for mocking, but neither is a lot of the other stuff in there and the less setup code in the tests the better, I reckon.